docs/.vuepress/head.js

```diff
@@ -1,4 +1,9 @@
 module.exports = [
+  [
+    'meta',
+    { name: 'viewport', content: 'width=device-width, initial-scale=1' }
+  ],
```

Comment:
```suggestion
```

This is already included in the [default SSR template](https://github.com/vuejs/vuepress/blob/master/packages/%40vuepress/core/lib/client/index.ssr.html).